Bit-Nation/BITNATION-Pangea-mobile
# [accounts] Password recovery - issue 480

#### Changes
- implemented reset password and reset pass code
- wrote the test for new functions

#### Verification
 -  Go to account then press Forgot password button
 -  Enter mnemonic to verify.
 -  Enter new password
 -  Navigate user to login screen after reset pass success
